csunplugged/templates/resources/binary-to-alphabet.html

```diff
@@ -0,0 +1,22 @@
+{% extends "resources/resource.html" %}
+
+<!-- Name of resource -->
+{% block name %}{{ resource.name }}{% endblock name %}
```

Comment:
This line is no longer required, check the updates to `resources/resource.html`.